sdk/keyvault/Azure.Security.KeyVault.Administration/src/RoleAssignmentScope.cs

```diff
@@ -28,10 +28,20 @@ public RoleAssignmentScope(string value)
         /// <summary>
         /// Initializes a new instance of the <see cref="RoleAssignmentScope"/> structure.
         /// </summary>
-        /// <param name="ResourceId">The Resource Id for the given Resource</param>
+        /// <param name="ResourceId">The Resource Id for the given Resource.</param>
         public RoleAssignmentScope(Uri ResourceId)
```

Comment:
After a quick glance below, I don't see unit tests for this. Should be easy to cover, especially with `TestCase`-attributed methods.